### ISSUE TITLE
[Catalog] Horizontal dots (⋯) overlap back button in AppBar when bold text is enabled

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4479]
When Accessibility Bold Text is enabled, 3 horizontal dots appear to overlap the back button icon in AppBars within the catalog.

Interestingly, the 3 dots are bolded, indicating that they might actually be text rather than an icon, since icons will not bold as a result of this setting.

It's also affecting buttons.

<img width="255" alt="screen shot 2018-07-06 at 12 57 23 pm" src="https://user-images.githubusercontent.com/1753199/42391105-35850da6-811c-11e8-9769-8e2b71c7209a.png">

## Reproduction steps
1. Open the Settings app and navigate to General &rarr; Accessibility &rarr; Bold Text. Set "Bold Text" to on. This may require the device to restart
1. Launch the MDC Catalog
1. Open any component demo list
1. Inspect the app bar

## Expected behavior
The "back" chevron is clearly visible.

![simulator screen shot - iphone 4s - 2018-07-06 at 12 38 29](https://user-images.githubusercontent.com/1753199/42390929-a1a5c5f8-811b-11e8-8bc7-936122a21ca5.png)

## Actual behavior
The "back" chevron is overlapped with horizontal dots.

![simulator screen shot - iphone 8 plus - 2018-07-06 at 12 46 54](https://user-images.githubusercontent.com/1753199/42390948-acb4982a-811b-11e8-9eba-751448809de4.png)

## Platform (please complete the following information)
iPhone 8+/iOS 11.0.1 (Simulator)

**Not reproducible on iPhone 4s/iOS 8.1 (Simulator)**

---
#### Internal data
- Associated internal bug: [b/117178211](http://b/117178211)

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
